ndarray-rand/src/lib.rs

```diff
@@ -73,18 +81,20 @@ impl<S, D> RandomExt<S, D> for ArrayBase<S, D>
           D: Dimension,
 {
     fn random<Sh, IdS>(shape: Sh, dist: IdS) -> ArrayBase<S, D>
-        where IdS: IndependentSample<S::Elem>,
+        where IdS: Distribution<S::Elem>,
               Sh: ShapeBuilder<Dim=D>,
     {
-        Self::random_using(shape, dist, &mut rand::weak_rng())
+        let mut rng = SmallRng::from_rng(thread_rng())
+            .unwrap_or_else(|err| panic!("create SmallRng from thread_rng failed: {:?}", err));
```

Comment:
Isn't this the same as `expect("create SmallRng from thread_rng failed")`? The way I understand the [documentation](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect), I believe it is equivalent. If so, I think we should prefer `expect`.

Reply:
Yes, you're right. I just made that change. Thanks.